internal/handlers/pg/msg_insert.go

```diff
@@ -98,6 +90,14 @@ func (h *Handler) MsgInsert(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 
 // insert prepares and executes actual INSERT request to Postgres.
 func (h *Handler) insert(ctx context.Context, sp sqlParam, doc any) error {
+	created, err := h.pgPool.CreateTableIfNotExists(ctx, sp.db, sp.collection)
```

Comment:
Moved this code here for consistency, as `fetch` has a similar construction. Also, `upsert` will have it too.